### PR TITLE
 layout_product-cards

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://SerhiiNalyvaiko.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://SerhiiNalyvaiko.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+        alt="APPLE A1419 iMac 27"
+        class="card__image image"
+      >
+      <h1 class="card__title title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__product-text product-text">
+        Product code: 195434
+      </p>
+      <div class="card__rewiews rewiew">
```
Comment (AlisaVlasova) [inline src/index.html:23]:
```suggestion
      <div class="card__rewiews">
```
you need only two BEM blocks in this task card and stars

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+          $2,199
+        </p>
+      </div>
+      <a
+        href="#"
+        class="card__button-link button-link"
+        data-qa="hover"
+      >
+        buy
+      </a>
```
Comment (AlisaVlasova) [inline src/index.html:47]:
```suggestion
        <h2 class="card__rewiew-title">Reviews: 5</h2>
      </div>
      <div class="card__price">
        <p class="card__price-text">
          Price:
        </p>
        <p class="card__price-total">
          $2,199
        </p>
      </div>
      <a
        href="#"
        class="card__button-link"
        data-qa="hover"
      >
        buy
      </a>
```

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]
check font color on the card and font-size on the button 
image

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
check styles, pls
you have different font size for button, different position of image, different colors of "Product code", "Price", different align of block with stars, and review

that's why we have test - you can check, what wrong eith your work

**Comment by SerhiiNalyvaiko** [comment]
sorry I didn't deploy

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**